src/Adapter/Driver/Sqlsrv/Connection.php

```diff
@@ -101,6 +101,7 @@ public function connect()
         $serverName = '.';
         $params     = [
             'ReturnDatesAsStrings' => true,
+            'TrustServerCertificate' => 1
```

Comment:
This is a security feature that can not be applied as a default for adapter. Please revert.
This connection parameter needs to be properly configured via `'options'` from test case.

Reply:
reverted